src/pages/contact.js

```diff
@@ -0,0 +1,72 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Head from 'next/head';
+
+import Footer from '../components/Footer';
+import HeaderContact from '../components/HeaderContact';
+
+const FormWrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 135px 50px 50px 50px;
+	padding: 30px;
+	border-radius: 30px;
+	background-color: white;
+	box-shadow: 0 0 18px -2px rgba(0, 0, 0, 0.07);
+	font-weight: 200;
+`;
+
+const SubmitButton = styled.button`
+	display: inline-block;
+	width: 100%;
+	height: 50px;
+	margin: 30px auto 0;
+	border: none;
+	border-radius: 30px;
+	background-color: #61adad;
+	color: white;
+`;
+
+export default function Contact() {
+	const handleSubmit = event => {
+		event.preventDefault();
+		const formdata = new FormData(event.target);
+		const data = Object.fromEntries(formdata);
+		console.log(data);
```

Comment:
Do you need this console.log?

Reply:
Thanks for review my code - The console.log is only for testing till I finished my feature. So I pushed my pull-request that it is not to long for review.